pages/en/lb3/Polymorphic-relations.md

```diff
@@ -45,7 +45,34 @@ The examples below use three example models: Picture, Author, and Reader, where
 
 ## HasMany polymorphic relations
 
-The usual options apply, for example: `as: 'photos'` to specify a different relation name/accessor.
+Take the following scenario as example:
+
+> A Picture model belongs to either an Author model or a Reader model 
+
+A hasMany polymorphic relation means: there are two properties created in model Picture, one serves as a discriminator, which tells which model the picture actually belongs to, and the other one servers as a foreignKey, which is the id(or primaryKey) value of either the author or reader.
```

Comment:
`which tells` is not clear, or grammatically correct. I'd rephrase this:
`A hasMany polymorphic relation means: there are two properties created in model Picture, one serves as a discriminator, which states what model the picture belongs to`

Additionally, you're using things like `belongs to` in a description about hasMany; it could be confusing for users that might think they can just use the two interchangably from either direction, rather than needing both for a two-way relationship.